src/compiler/diagnosticMessages.json

```diff
@@ -1859,10 +1859,22 @@
         "category": "Error",
         "code": 2553
     },
-    "Expected {0} type arguments, but got {1}.": {
+    "Expected at least {0} arguments, but got {1}.": {

```

Comment:
`"Expected {0} type arguments, but got {1}."` is closer to the old message. I'd argue that the error code didn't need to change. Consider switching this with the message assigned to code 2557